runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -444,12 +455,12 @@ private void translateS3AndKinesisInputs(
       EventConfig.Builder builder = new EventConfig.Builder(node.get("name").asText());
 
       // Add lateness in sec
-      JsonNode latenessInSec = node.get("lateness_in_sec");
-      if (latenessInSec != null) {
-        builder = builder.withLatenessInSec(latenessInSec.asLong());
+      JsonNode maxOutOfOrdernessMillis = node.get("max_out_of_orderness_millis");
+      if (maxOutOfOrdernessMillis != null) {
+        builder = builder.withLatenessInSec(maxOutOfOrdernessMillis.asLong()/1000);
```

Comment:
Lets just add a TODO here to change the builder method name later as well.

Reply:
I made the change to avoid tripping off Beam users. One is developer friendly and the other is streaming pro friendly :) Added a TODO for now.